### PR TITLE
log-backup: add known issue: azure or gcs first truncate

### PR DESCRIPTION
**Pull request by Leavrth**
Signed-off-by: Leavrth <jianjun.liao@outlook.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
add a known issue

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [x] v6.3 (TiDB 6.3 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/pitr-known-issues.md [hunk 1]
```diff
@@ -66,3 +66,12 @@ Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
...
 - 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份增加的索引数据。
+
+## 在 GCS 或 Azure Blob Storage 上第一次执行 PITR Truncate 时出错
+
+Issue 链接：[#38229](https://github.com/pingcap/tidb/issues/38229)
+
+在 GCS 或 Azure Blob Storage 上**第一次**执行 PITR Truncate 时会提示文件 `v1_stream_trancate_safepoint.txt` 不存在。有以下两种解决方法：
+
+1. 在 PITR 的备份根目录下创建文件 `v1_stream_trancate_safepoint.txt`，文件内容为 `0`。注意没有别的额外字符，并且只需要在第一次做 truncate 命令时添加。
+2. 使用新版本的 BR。
```
Comment (3pointer) [inline br/pitr-known-issues.md:77]: 需要明确下版本

File: br/pitr-known-issues.md [hunk 2]
```diff
@@ -66,3 +66,13 @@ Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
...
+
+## 在 GCS 或 Azure Blob Storage 上第一次执行 PITR Truncate 时出错
+
+Issue 链接：[#38229](https://github.com/pingcap/tidb/issues/38229)
+
+在 GCS 或 Azure Blob Storage 上**第一次**执行 PITR Truncate 时会提示文件 `v1_stream_trancate_safepoint.txt` 不存在。有以下解决方法：
+
+* 在 PITR 的备份根目录下创建文件 `v1_stream_trancate_safepoint.txt`，文件内容为 `0`。注意没有别的额外字符，并且只需要在第一次做 truncate 命令时添加。
+<!-- TODO: v6.4.0 发版时，取消注释以下文字  -->
+<!-- 2. 使用 v6.4.0 及以上版本的 BR。 -->
```
Comment (shichun-0415) [inline br/pitr-known-issues.md:78]:
```suggestion
在 GCS 或 Azure Blob Storage 上**第一次**执行 PITR Truncate 时会提示文件 `v1_stream_trancate_safepoint.txt` 不存在。解决方法如下：

在 PITR 的备份根目录下，创建内容为 `0` 的文件 `v1_stream_trancate_safepoint.txt`。注意文件不得包含其他字符，并且只需要在第一次做 truncate 命令时添加。
<!-- TODO: v6.4.0 发版时，取消注释以下文字  -->
<!-- 或者，使用 v6.4.0 及以上版本的 BR。 -->
```

File: br/pitr-known-issues.md [hunk 3]
```diff
@@ -66,3 +66,13 @@ Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
 - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
 - 如果先启动添加索引加速任务，再创建 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
 - 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份增加的索引数据。
+
+## 在 GCS 或 Azure Blob Storage 上第一次执行 PITR Truncate 时出错
```
Comment (shichun-0415) [inline br/pitr-known-issues.md:70]:
```suggestion
## 在 GCS 或 Azure Blob Storage 上第一次执行 `PITR Truncate` 时出错
```

File: br/pitr-known-issues.md [hunk 4]
```diff
@@ -66,3 +66,13 @@ Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
...
+
+Issue 链接：[#38229](https://github.com/pingcap/tidb/issues/38229)
+
+在 GCS 或 Azure Blob Storage 上**第一次**执行 PITR Truncate 时会提示文件 `v1_stream_trancate_safepoint.txt` 不存在。解决方法如下：
+
+在 PITR 的备份根目录下，创建内容为 `0` 的文件 `v1_stream_trancate_safepoint.txt`。注意文件不得包含其他字符，并且只需要在第一次做 truncate 命令时添加。
```
Comment (shichun-0415) [inline br/pitr-known-issues.md:76]:
```suggestion
在 PITR 的备份根目录下，创建内容为 `0` 的文件 `v1_stream_trancate_safepoint.txt`。注意文件不得包含其他字符，并且只需要在第一次执行 `PITR Truncate` 命令时添加。
```

**Review by shichun-0415 — APPROVED** [review]

File: br/pitr-known-issues.md [hunk 5]
```diff
@@ -66,3 +66,13 @@ Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
 - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
 - 如果先启动添加索引加速任务，再创建 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
 - 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份增加的索引数据。
+
+## 在 GCS 或 Azure Blob Storage 上第一次执行 `PITR Truncate` 时出错
+
+Issue 链接：[#38229](https://github.com/pingcap/tidb/issues/38229)
+
+在 GCS 或 Azure Blob Storage 上**第一次**执行 PITR Truncate 时会提示文件 `v1_stream_trancate_safepoint.txt` 不存在。解决方法如下：
```
Comment (shichun-0415) [inline br/pitr-known-issues.md:74]:
```suggestion
在 GCS 或 Azure Blob Storage 上**第一次**执行 `PITR Truncate` 时会提示文件 `v1_stream_trancate_safepoint.txt` 不存在。解决方法如下：
```

File: br/pitr-known-issues.md [hunk 6]
```diff
@@ -66,3 +66,13 @@ Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
 - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
 - 如果先启动添加索引加速任务，再创建 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
 - 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份增加的索引数据。
+
+## 在 GCS 或 Azure Blob Storage 上第一次执行 `PITR Truncate` 时出错
+
+Issue 链接：[#38229](https://github.com/pingcap/tidb/issues/38229)
```
Comment (TomShawn) [inline br/pitr-known-issues.md:72]:
```suggestion
相关 GitHub Issue：[#38229](https://github.com/pingcap/tidb/issues/38229)
```

File: br/pitr-known-issues.md [hunk 7]
```diff
@@ -66,3 +66,13 @@ Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
...
+
+在 GCS 或 Azure Blob Storage 上**第一次**执行 `PITR Truncate` 时会提示文件 `v1_stream_trancate_safepoint.txt` 不存在。解决方法如下：
+
+在 PITR 的备份根目录下，创建内容为 `0` 的文件 `v1_stream_trancate_safepoint.txt`。注意文件不得包含其他字符，并且只需要在第一次执行 `PITR Truncate` 命令时添加。
```
Comment (TomShawn) [inline br/pitr-known-issues.md:76]: 多少个 0 呢？
```suggestion
在 PITR 的备份根目录下，创建文件 `v1_stream_trancate_safepoint.txt`，文件中仅包含字符 `0`。注意文件不得包含其他字符，并且只需要在第一次执行 `PITR Truncate` 命令时添加。
```

File: br/pitr-known-issues.md [hunk 8]
```diff
@@ -66,3 +66,13 @@ Issue 链接：[#38045](https://github.com/pingcap/tidb/issues/38045)
 - 如果先启动 PITR 备份任务，再添加索引，此时即使索引加速功能打开，也不会使用加速索引功能，但不影响索引兼容性。
 - 如果先启动添加索引加速任务，再创建 PITR 备份任务，此时 PITR 备份任务会报错，但不影响正在添加索引的任务。
 - 如果同时启动 PITR 备份任务和添加索引加速任务，可能会由于两个任务无法察觉到对方而导致 PITR 不能成功备份增加的索引数据。
+
+## 在 GCS 或 Azure Blob Storage 上第一次执行 `PITR Truncate` 时出错
```
Comment (TomShawn) [inline br/pitr-known-issues.md:70]:
```suggestion
## 在 GCS 或 Azure Blob Storage 上第一次执行 `PITR Truncate` 命令时出错
```

**Review by TomShawn — APPROVED** [review]

File: br/pitr-known-issues.md (hunk 6 above)
Comment (shichun-0415) [reply br/pitr-known-issues.md:72]: 本文其他地方都是 Issue 链接，所以我认为可以跟基线保持一致，另外，打开 issue 能看到是 GitHub 的 issue，不加说明我觉得没问题。

File: br/pitr-known-issues.md (hunk 7 above)
Comment (shichun-0415) [reply br/pitr-known-issues.md:76]: 创建文件 `v1_stream_trancate_safepoint.txt`，文件中仅包含字符 `0`，改成
> 创建文件 `v1_stream_trancate_safepoint.txt`，然后写入 `0`。

Comment (TomShawn) [reply br/pitr-known-issues.md:76]: LGTM

File: br/pitr-known-issues.md (hunk 6 above)
Comment (TomShawn) [reply br/pitr-known-issues.md:72]: LGTM

**Comment by IANTHEREAL** [comment]
LGTM

**Review by IANTHEREAL — APPROVED** [review]

**Pull request merged**